services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_api.py

```diff
@@ -427,3 +428,84 @@ async def is_dynamic_service_running(
         )
 
         return len(dynamic_sidecar_services) == 1
+
+
+async def get_or_create_networks_ids(
+    networks: List[str], project_id: ProjectID
+) -> Dict[str, str]:
+    async def _get_id_from_name(client, network_name: str) -> str:
+        network = await client.networks.get(network_name)
```

Comment:
if the network is not found it will raise here.

Reply:
Had a look at the function. It should not happen since the network should be created at the previous step. But in the remote case it will happen. A generic aiodocker error will be raised, which is fine.